examples/sktime/flavor.py

```diff
@@ -0,0 +1,639 @@
+
+        For more information about supported URI schemes, see
+        `Referencing Artifacts <https://www.mlflow.org/docs/latest/tracking.html#
+        artifact-locations>`_.
+    dst_path : str, optional (default=None)
+        The local filesystem path to which to download the model artifact.This
+        directory must already exist. If unspecified, a local output path will
+        be created.
+
+    Returns
+    -------
+    A sktime model instance.
+    """
+    local_model_path = _download_artifact_from_uri(artifact_uri=model_uri, output_path=dst_path)
+    flavor_conf = _get_flavor_configuration(model_path=local_model_path, flavor_name=FLAVOR_NAME)
+    _add_code_from_conf_to_system_path(local_model_path, flavor_conf)
+    sktime_model_file_path = os.path.join(local_model_path, flavor_conf["pickled_model"])
+    serialization_format = flavor_conf.get("serialization_format", SERIALIZATION_FORMAT_PICKLE)
+    return _load_model(path=sktime_model_file_path, serialization_format=serialization_format)
+
+
+def _save_model(model, path, serialization_format):
+    with open(path, "wb") as out:
+        if serialization_format == SERIALIZATION_FORMAT_PICKLE:
+            pickle.dump(model, out)
+        elif serialization_format == SERIALIZATION_FORMAT_CLOUDPICKLE:
+            import cloudpickle
+
+            cloudpickle.dump(model, out)
+        else:
+            raise MlflowException(
+                message="Unrecognized serialization format: "
+                "{serialization_format}".format(serialization_format=serialization_format),
+                error_code=INTERNAL_ERROR,
+            )
+
+
+def _load_model(path, serialization_format):
+    if serialization_format not in SUPPORTED_SERIALIZATION_FORMATS:
+        raise MlflowException(
+            message=(
+                "Unrecognized serialization format: {serialization_format}. "
+                "Please specify one of the following supported formats: "
+                "{supported_formats}.".format(
+                    serialization_format=serialization_format,
+                    supported_formats=SUPPORTED_SERIALIZATION_FORMATS,
+                )
+            ),
+            error_code=INVALID_PARAMETER_VALUE,
+        )
+
+    with open(path, "rb") as pickled_model:
+        if serialization_format == SERIALIZATION_FORMAT_PICKLE:
+            return pickle.load(pickled_model)
+        elif serialization_format == SERIALIZATION_FORMAT_CLOUDPICKLE:
+            import cloudpickle
+
+            return cloudpickle.load(pickled_model)
+
+
+def _load_pyfunc(path):
+    """Load PyFunc implementation. Called by ``pyfunc.load_model``.
+
+    Parameters
+    ----------
+    path : str
+        Local filesystem path to the MLflow Model with the sktime flavor.
+
+    """
+    if os.path.isfile(path):
+        serialization_format = SERIALIZATION_FORMAT_PICKLE
+        _logger.warning("Loading procedure in older versions of MLflow using pickle.load()")
```

Comment:
? Older versions of MLflow? This only exists as of when this PR is being filed. Where is this warning coming from?

Reply:
was taken from `def _load_pyfunc(path)` here https://github.com/mlflow/mlflow/blob/master/mlflow/sklearn/__init__.py, but does not make sense...

will removet those lines 458-460